shell/platform/windows/testing/engine_embedder_api_modifier.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/flutter_windows_engine.h"
+
+namespace flutter {
+
+// A test utility class providing the ability to access and alter the embedder
+// API proc table for an engine instance.
```

Comment:
nit: Can you add a comment here saying that any modifications will outlive the lifetime of this object?

Reply:
Done; added both here and to embedder_api() just to make sure it's hard to miss since given the oddity of the API.